docs/blog/newsletter-july-2021.rst

```diff
@@ -0,0 +1,122 @@
+
+.. post:: July 06, 2021
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2021
+#####################################
+
+Good morning, afternoon, or evening, documentarians! It's Beth and the newsletter team with a last summer update before we head off on a little aestival break. We'll be back in September :) 
```

Comment:
I had to double check `aestival` which probably means we should rework it ;-)

Reply:
yeah, I was being silly with collateral adjectives. removed.